csharp/lib/ghsl/ConstExpressions.qll

```diff
@@ -0,0 +1,67 @@
+import csharp
```

Comment:
I would be very nice, if this module is imported in https://github.com/GitHubSecurityLab/CodeQL-Community-Packs/blob/main/csharp/src/security/JsonWebTokenHandler/JsonWebTokenHandlerLib.qll and then the code duplication can be deleted.